tests/drivers/flash/conf/nrf52840_flash_sdc.conf

```diff
@@ -0,0 +1,9 @@
+# configuration for testing softdevice controller
+# flash driver on nrf52840dk_nrf52840 board
```

Comment:
I would suggest to remove `nrf52840dk_nrf52840` from these files and the file name. These configurations are generic and can be reused by other boards as well